# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmModule.cpp
# Commit: 538920d2d777
# Full Hash: 538920d2d77751743a51f10cf730f1ce65b5d0ee
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2020-08-18 09:24:52
# Regressor Bug: 1561521
# File Overlap Count: 2
# Description:
#   Bug 1561521 - Rename TypeCode/ValType/RefType::Any to Extern. r=lth
#   
#   This commit renames the TypeCode/ValType/RefType occurrences of 'any' to 'extern'
#   to match what they actually represent. All public API facing occurrences have
#   already been switched to 'externref' so this should only be aligning the
# ==============================================================================

diff -r 3d3e3d124767 -r 538920d2d777 js/src/wasm/WasmModule.cpp
--- a/js/src/wasm/WasmModule.cpp	Tue Aug 18 00:03:29 2020 +0000
+++ b/js/src/wasm/WasmModule.cpp	Tue Aug 18 00:04:37 2020 +0000
@@ -1209,7 +1209,7 @@
               cx, cx->global(), ReferenceType::TYPE_OBJECT);
           break;
         case RefType::Func:
-        case RefType::Any:
+        case RefType::Extern:
           t = GlobalObject::getOrCreateReferenceTypeDescr(
               cx, cx->global(), ReferenceType::TYPE_WASM_ANYREF);
           break;